game/pitch.hh

```diff
@@ -117,6 +85,11 @@ public:
 	/** Returns the id (color name) of the mic */
 	std::string const& getId() const { return m_id; }
 
+private:
+	bool calcFFT();
+	void calcTones();
+	void mergeWithOld(tones_t& tones) const;
+
```

Comment:
please do not define double private sections
Do keep the methods together and fields together though

I'd prefer the order of method - member